### PR TITLE
Update use-display-requirements-llm.md

### PR DESCRIPTION
**Pull request by Mohammed-Deifallah**
This page needs to be updated in order to reflect the new feature of Bing Custom Search with LLM. For more information, take a look on this task: https://msasg.visualstudio.com/Bing_UX/_workitems/edit/7737539

### CODE REVIEW

**Comment by GhiathAjam** [comment]
Please resolve the warning

File: bing-docs/bing-web-search/use-display-requirements-llm.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ These Bing Search APIs, with your LLM use and display requirements apply to any
 - Bing News Search
 - Bing Video Search
 - Bing Web Search
+- Bing Custom Search
 - Bing Spell Check
 - Bing Autosuggest
```
Comment (GhiathAjam) [inline bing-docs/bing-web-search/use-display-requirements-llm.md:30]: `Note use of Bing Custom Search and Bing Visual Search is not included in Bing Search APIs, with your LLM.
`

Looks like you need to update this too.

Comment (GhiathAjam) [reply bing-docs/bing-web-search/use-display-requirements-llm.md:30]: Line 30 | 31

**Review by GhiathAjam — CHANGES_REQUESTED** [review]

File: bing-docs/bing-web-search/use-display-requirements-llm.md (hunk 1 above)
Comment (Mohammed-Deifallah) [reply bing-docs/bing-web-search/use-display-requirements-llm.md:30]: Yes, that's right.
Thanks a lot for your catch

**Review by GhiathAjam — APPROVED** [review]

**Comment by Mohammed-Deifallah** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by GhiathAjam** [comment]
Congratulations!

**Pull request merged**